docs/TAGS.md

```diff
@@ -93,7 +93,13 @@ Description: To generate secrets protection masterkey if secrets protection enab
 
 ### Tag - package
 
-Description: This is applied to all package installation related tasks. Skip this if you don't want to reinstall.
+Description: This is applied for all common packages installation tasks [like Java, openssl, gnupg, pip, etc]. Skip this if you don't want to reinstall.
```

Comment:
packages would be superset of all common and CP packages

Reply:
yes, thanks @nsharma-git , re-worded the doc.